install_prototool.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
Can you use python scripts instead? We are basically getting rid of .sh from the entire codebase.

Reply:
Sure.

Reply:
Done